components/admin-users.tsx

```
@@ -1,369 +1,232 @@
+    setLoading(true);
+    setError(null);
+
     try {
-      const url = editingUser ? `/api/users/${editingUser.id}` : "/api/users";
       const method = editingUser ? "PUT" : "POST";
+      const url = editingUser ? `/api/users/${editingUser.id}` : "/api/users";
+      const body: any = { ...formData };
+
+      // Remove password if editing and left blank
+      if (editingUser && !formData.password) {
+        delete body.password;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)